src/openrct2/drawing/ImageImporter.cpp

```diff
@@ -388,17 +390,23 @@ namespace OpenRCT2::Drawing
         auto yOffset = Json::GetNumber<int16_t>(input["y"]);
         auto keepPalette = Json::GetString(input["palette"]) == "keep";
         auto palette = keepPalette ? Palette::KeepIndices : Palette::OpenRCT2;
-
+        auto flags = EnumValue(ImportFlags::None);
         auto raw = Json::GetString(input["format"]) == "raw";
-        auto flags = raw ? ImportFlags::None : ImportFlags::RLE;
-
+        if (raw)
+            flags |= EnumValue(ImportFlags::RLE);
```

Comment:
Could you add some braces around this for legibility? i.e.
```suggestion
        if (raw)
        {
            flags |= EnumValue(ImportFlags::RLE);
        }
```

Reply:
We generally don’t do that. I do think a blank line before and after is a good thing, though.